cmd/kubeadm/app/cmd/phases/join/controlplane.go

```diff
@@ -182,59 +85,37 @@ func newControlPlanePrepareKubeconfigSubphase() workflow.Phase {
 		Name:         "kubeconfig",
 		Short:        "Generates the kubeconfig for the new control plane components",
 		Run:          runControlPlanePrepareKubeconfigPhaseLocal,
-		InheritFlags: getControlPlanePreparePhaseFlags(),
+		InheritFlags: getControlPlanePreparePhaseFlags(), //NB. eventually in future we would like to break down this in sub phases for each kubeconfig
 	}
 }
 
 func newControlPlanePrepareManifestsSubphases() workflow.Phase {
 	return workflow.Phase{
-		Name:  "manifests",
-		Short: "Generates the manifests for the new control plane components",
-		Phases: []workflow.Phase{
-			{
-				Name:           "all",
-				Short:          "Generates all static Pod manifest files",
-				InheritFlags:   getControlPlanePreparePhaseFlags(),
-				RunAllSiblings: true,
-			},
-			newControlPlanePrepareSubphase(kubeadmconstants.KubeAPIServer),
-			newControlPlanePrepareSubphase(kubeadmconstants.KubeControllerManager),
-			newControlPlanePrepareSubphase(kubeadmconstants.KubeScheduler),
-		},
-		InheritFlags: getControlPlanePreparePhaseFlags(),
-	}
-}
-
-func newControlPlanePrepareSubphase(component string) workflow.Phase {
-	return workflow.Phase{
-		Name:         controlPlanePhaseProperties[component].name,
-		Short:        controlPlanePhaseProperties[component].short,
-		Run:          runControlPlanePrepareJoinSubphase(component),
-		InheritFlags: getControlPlanePreparePhaseFlags(),
+		Name:         "manifests",
+		Short:        "Generates the manifests for the new control plane components",
+		Run:          runControlPlaneSubphase,
+		InheritFlags: getControlPlanePreparePhaseFlags(), //NB. eventually in future we would like to break down this in sub phases for each component
```

Comment:
Why lose the subphases here and add the comment to add the subphases in the future?